### PR TITLE
ci: Standardize formatting of Bazel command

### PR DESCRIPTION
**Pull request by WillEngFlow**
Signed-off-by: Will Martin <will@engflow.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description:
Move all Bazel arguments onto newlines to improve readability.

This is in preparation for making "remote-ci" arguments conditional based on the presence of `GITHUB_TOKEN`

Risk Level: Low
Testing: Manual
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

File: .github/workflows/cc_tests.yml [hunk 1]
```diff
@@ -22,4 +22,9 @@ jobs:
       - env:
             GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         name: 'Run tests'
-        run: ./bazelw test --action_env=LD_LIBRARY_PATH --test_output=all --config=remote-ci-linux --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/cc/...
+        run: ./bazelw test \
+          --action_env=LD_LIBRARY_PATH \
+          --test_output=all \
+          --config=remote-ci-linux \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/cc/...
```
Comment (anfelbar) [inline .github/workflows/cc_tests.yml:30]:
```suggestion
        run: |
          ./bazelw test \
          --action_env=LD_LIBRARY_PATH \
          --test_output=all \
          --config=remote-ci-linux \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/cc/...
```

File: .github/workflows/core.yml [hunk 2]
```diff
@@ -20,4 +20,8 @@ jobs:
       - name: 'Run tests'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw test --test_output=all --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN"  //test/common/...
+        run: ./bazelw test \
+          --test_output=all \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+           //test/common/...
```
Comment (anfelbar) [inline .github/workflows/core.yml:27]:
```suggestion
        run: |
          ./bazelw test \
          --test_output=all \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
           //test/common/...
```

File: .github/workflows/ios_build.yml [hunk 3]
```diff
@@ -36,12 +40,20 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/hello_world:app
+        run: ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/hello_world:app
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:47]:
```suggestion
        run: |
          ./bazelw build \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/swift/hello_world:app
```

File: .github/workflows/ios_build.yml [hunk 4]
```diff
@@ -36,12 +40,20 @@ jobs:
...
         name: 'Build swift app'
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/hello_world:app &> /tmp/envoy.log &
+        run: ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/hello_world:app &> /tmp/envoy.log &
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:56]:
```suggestion
        run: |
          ./bazelw run \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/swift/hello_world:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 5]
```diff
@@ -61,12 +73,20 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/baseline:app
+        run: ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/baseline:app
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:80]:
```suggestion
        run: |
          ./bazelw build \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/swift/apps/baseline:app
```

File: .github/workflows/ios_build.yml [hunk 6]
```diff
@@ -61,12 +73,20 @@ jobs:
...
         name: 'Build swift app'
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/baseline:app &> /tmp/envoy.log &
+        run: ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/baseline:app &> /tmp/envoy.log &
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:89]:
```suggestion
        run: |
          ./bazelw run \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/swift/apps/baseline:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 7]
```diff
@@ -86,12 +106,20 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/experimental:app
+        run: ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/experimental:app
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:113]:
```suggestion
        run: |
          ./bazelw build \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/swift/apps/experimental:app
```

File: .github/workflows/ios_build.yml [hunk 8]
```diff
@@ -86,12 +106,20 @@ jobs:
...
         name: 'Build swift app'
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/experimental:app &> /tmp/envoy.log &
+        run: ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/experimental:app &> /tmp/envoy.log &
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:122]:
```suggestion
        run: |
          ./bazelw run \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/swift/apps/experimental:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 9]
```diff
@@ -111,12 +139,20 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/async_await:app
+        run: ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/async_await:app
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:146]:
```suggestion
        run: |
          ./bazelw build \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/swift/async_await:app
```

File: .github/workflows/ios_build.yml [hunk 10]
```diff
@@ -111,12 +139,20 @@ jobs:
...
         name: 'Build swift app'
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/async_await:app &> /tmp/envoy.log &
+        run: ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/async_await:app &> /tmp/envoy.log &
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:155]:
```suggestion
        run: |
          ./bazelw run \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/swift/async_await:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 11]
```diff
@@ -136,12 +172,20 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/objective-c/hello_world:app
+        run: ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/objective-c/hello_world:app
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:179]:
```suggestion
        run: |
          ./bazelw build \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/objective-c/hello_world:app
```

File: .github/workflows/ios_build.yml [hunk 12]
```diff
@@ -136,12 +172,20 @@ jobs:
...
         name: 'Build objective-c app'
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/objective-c/hello_world:app &> /tmp/envoy.log &
+        run: ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/objective-c/hello_world:app &> /tmp/envoy.log &
```
Comment (anfelbar) [inline .github/workflows/ios_build.yml:188]:
```suggestion
        run: |
          ./bazelw run \
          --config=ios \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //examples/objective-c/hello_world:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_tests.yml [hunk 13]
```diff
@@ -31,7 +31,14 @@ jobs:
...
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw test --experimental_ui_max_stdouterr_bytes=10485760 --test_output=all --config=ios --build_tests_only --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/...
+        run: ./bazelw test \
+          --experimental_ui_max_stdouterr_bytes=10485760 \
+          --test_output=all \
+          --config=ios \
+          --build_tests_only \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/...
```
Comment (anfelbar) [inline .github/workflows/ios_tests.yml:41]:
```suggestion
        run: |
          ./bazelw test \
          --experimental_ui_max_stdouterr_bytes=10485760 \
          --test_output=all \
          --config=ios \
          --build_tests_only \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/swift/...
```

File: .github/workflows/ios_tests.yml [hunk 14]
```diff
@@ -56,4 +63,11 @@ jobs:
...
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw test --test_output=all --config=ios --build_tests_only --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/objective-c/...  //test/cc/unit:envoy_config_test
+        run: ./bazelw test \
+          --test_output=all \
+          --config=ios \
+          --build_tests_only \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/objective-c/...  \
+          //test/cc/unit:envoy_config_test
```
Comment (anfelbar) [inline .github/workflows/ios_tests.yml:73]:
```suggestion
        run: |
          ./bazelw test \
          --test_output=all \
          --config=ios \
          --build_tests_only \
          --config=remote-ci-macos \
          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
          //test/objective-c/...  \
          //test/cc/unit:envoy_config_test
```

File: .github/workflows/tsan.yml [hunk 15]
```diff
@@ -42,8 +42,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: steps.check_context.outputs.run_tests == 'true'
-        run: |
-          ./bazelw test --test_output=all \
+        run: ./bazelw test \
+          --test_output=all \
```
Comment (anfelbar) [inline .github/workflows/tsan.yml:46]:
```suggestion
        run: 
          ./bazelw test \
          --test_output=all \
```

**Review by anfelbar — DISMISSED** [review]

**Comment by anfelbar** [comment]
@jpsim can you please review this PR, it is ready :)
Thank you.

File: .github/workflows/core.yml [hunk 16]
```diff
@@ -20,4 +20,9 @@ jobs:
       - name: 'Run tests'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw test --test_output=all --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN"  //test/common/...
+        run: |
+          ./bazelw test \
+          --test_output=all \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+           //test/common/...
```
Comment (jpsim) [inline .github/workflows/core.yml:28]: Nit:

```suggestion
          //test/common/...
```

File: .github/workflows/ios_build.yml [hunk 17]
```diff
@@ -36,12 +40,22 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/hello_world:app
+        run: |
+          ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/hello_world:app
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:48]: Nit:
```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/swift/hello_world:app
```

File: .github/workflows/ios_build.yml [hunk 18]
```diff
@@ -36,12 +40,22 @@ jobs:
...
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/hello_world:app &> /tmp/envoy.log &
+        run: |
+          ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/hello_world:app &> /tmp/envoy.log &
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:58]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/swift/hello_world:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 19]
```diff
@@ -61,12 +75,22 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/baseline:app
+        run: |
+          ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/baseline:app
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:83]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/swift/apps/baseline:app
```

File: .github/workflows/ios_build.yml [hunk 20]
```diff
@@ -61,12 +75,22 @@ jobs:
...
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/baseline:app &> /tmp/envoy.log &
+        run: |
+          ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/baseline:app &> /tmp/envoy.log &
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:93]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/swift/apps/baseline:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 21]
```diff
@@ -86,12 +110,22 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/experimental:app
+        run: |
+          ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/experimental:app
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:118]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/swift/apps/experimental:app
```

File: .github/workflows/ios_build.yml [hunk 22]
```diff
@@ -86,12 +110,22 @@ jobs:
...
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/apps/experimental:app &> /tmp/envoy.log &
+        run: |
+          ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/apps/experimental:app &> /tmp/envoy.log &
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:128]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/swift/apps/experimental:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 23]
```diff
@@ -111,12 +145,22 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/async_await:app
+        run: |
+          ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/async_await:app
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:153]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/swift/async_await:app
```

File: .github/workflows/ios_build.yml [hunk 24]
```diff
@@ -111,12 +145,22 @@ jobs:
...
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/swift/async_await:app &> /tmp/envoy.log &
+        run: |
+          ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/swift/async_await:app &> /tmp/envoy.log &
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:163]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/swift/async_await:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_build.yml [hunk 25]
```diff
@@ -136,12 +180,22 @@ jobs:
         name: 'Install dependencies'
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw build --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/objective-c/hello_world:app
+        run: |
+          ./bazelw build \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/objective-c/hello_world:app
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:188]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/objective-c/hello_world:app
```

File: .github/workflows/ios_build.yml [hunk 26]
```diff
@@ -136,12 +180,22 @@ jobs:
...
       # Run the app in the background and redirect logs.
       - env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: ./bazelw run --config=ios --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //examples/objective-c/hello_world:app &> /tmp/envoy.log &
+        run: |
+          ./bazelw run \
+          --config=ios \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //examples/objective-c/hello_world:app &> /tmp/envoy.log &
```
Comment (jpsim) [inline .github/workflows/ios_build.yml:198]: Nit:

```suggestion
            --config=ios \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //examples/objective-c/hello_world:app &> /tmp/envoy.log &
```

File: .github/workflows/ios_tests.yml [hunk 27]
```diff
@@ -31,7 +31,15 @@ jobs:
...
-        run: ./bazelw test --experimental_ui_max_stdouterr_bytes=10485760 --test_output=all --config=ios --build_tests_only --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/swift/...
+        run: |
+          ./bazelw test \
+          --experimental_ui_max_stdouterr_bytes=10485760 \
+          --test_output=all \
+          --config=ios \
+          --build_tests_only \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/swift/...
```
Comment (jpsim) [inline .github/workflows/ios_tests.yml:42]: Nit:

```suggestion
            --experimental_ui_max_stdouterr_bytes=10485760 \
            --test_output=all \
            --config=ios \
            --build_tests_only \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/swift/...
```

File: .github/workflows/ios_tests.yml [hunk 28]
```diff
@@ -56,4 +64,12 @@ jobs:
...
-        run: ./bazelw test --test_output=all --config=ios --build_tests_only --config=remote-ci-macos --remote_header="Authorization=Bearer $GITHUB_TOKEN" //test/objective-c/...  //test/cc/unit:envoy_config_test
+        run: |
+          ./bazelw test \
+          --test_output=all \
+          --config=ios \
+          --build_tests_only \
+          --config=remote-ci-macos \
+          --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
+          //test/objective-c/...  \
+          //test/cc/unit:envoy_config_test
```
Comment (jpsim) [inline .github/workflows/ios_tests.yml:75]: Nit:

```suggestion
            --test_output=all \
            --config=ios \
            --build_tests_only \
            --config=remote-ci-macos \
            --remote_header="Authorization=Bearer $GITHUB_TOKEN" \
            //test/objective-c/...  \
            //test/cc/unit:envoy_config_test
```

File: .github/workflows/tsan.yml [hunk 29]
```diff
@@ -43,7 +43,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: steps.check_context.outputs.run_tests == 'true'
         run: |
-          ./bazelw test --test_output=all \
+          ./bazelw test \
+          --test_output=all \
```
Comment (jpsim) [inline .github/workflows/tsan.yml:47]: Nit: indentation in these lines too

**Review by jpsim — DISMISSED** [review]
If we're gonna change this, might as well consistently indent following lines.

Ping me when that's done and I'll re-approve and merge.

File: .github/workflows/tsan.yml (hunk 29 above)
Comment (WillEngFlow) [reply .github/workflows/tsan.yml]: Done.

File: local_tsan.bazelrc [hunk 30]
```diff
@@ -0,0 +1,6 @@
+# Generated file, do not edit. Delete this file if you no longer use local tsan-instrumented libc++
+build:local-tsan --config=libc++
```
Comment (jpsim) [inline local_tsan.bazelrc:2]: Checked in by mistake?

**Comment by jpsim** [comment]
Looks good now other than `local_tsan.bazelrc`.

File: local_tsan.bazelrc (hunk 30 above)
Comment (WillEngFlow) [reply local_tsan.bazelrc:2]: Oops, thought I reverted that

**Review by jpsim — APPROVED** [review]

**Pull request merged**